[
  {
    "DefinitionID": "def-main-v3.9-CVE-2018-16395",
    "Title": "CVE-2018-16395",
    "Description": "",
    "Advisory": {
      "Severity": "",
      "Cves": [
        {
          "CveID": "CVE-2018-16395",
          "Cvss2": "",
          "Cvss3": "",
          "Cwe": "",
          "Impact": "",
          "Href": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-16395",
          "Public": ""
        }
      ],
      "Bugzillas": [],
      "AffectedCPEList": [],
      "Issued": "1000-01-01T00:00:00Z",
      "Updated": "1000-01-01T00:00:00Z"
    },
    "Debian": null,
    "AffectedPacks": [
      {
        "Name": "ruby",
        "Version": "2.5.2-r0",
        "Arch": "",
        "NotFixedYet": false,
        "ModularityLabel": ""
      }
    ],
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2018-16395",
        "RefURL": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-16395"
      }
    ]
  }
]
